client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -87,12 +87,12 @@ class InlineHelpSearchCard extends Component {
 			} );
 		}
 
-		// Make a search
-		this.props.requestInlineHelpSearchResults( searchQuery );
+		// Set the query search
+		this.props.setInlineHelpSearchQuery( searchQuery );
 	};
 
 	componentDidMount() {
-		this.props.requestInlineHelpSearchResults();
+		this.props.setInlineHelpSearchQuery();
```

Comment:
> I shouldn't have to know that setInlineHelpSearchQuery has a default argument.

Debatable :-D. Can we pass the empty string straight to the function argument instead of creating a new const?

Reply:
```es6
this.props.setInlineHelpSearchQuery( '' );
```